R/toolStatusMessage.R

```diff
@@ -25,6 +25,9 @@
 toolStatusMessage <- function(status, message, level = 0) {
   symbol <- toolSubtypeSelect(status, c(ok = "\u2713", note = "!", warn = "WARNING"))
   message <- paste0("[", symbol, "] ", message)
-  vcat(ifelse(status == "warn", 0,1), message, show_prefix = FALSE)
-  putMadratMessage("status", message, fname = -2 - level, add = TRUE)
+  vcat(ifelse(status == "warn", 0, 1), message, show_prefix = FALSE)
```

Comment:
While we're at it:
```suggestion
  vcat(if (status == "warn") 0 else 1, message, show_prefix = FALSE)
```

> if(test) yes else no is much more efficient and often much preferable to ifelse(test, yes, no) whenever test is a simple true/false result, i.e., when length(test) == 1

https://rdrr.io/r/base/ifelse.html